src/etc/lldb_lookup.py

```diff
@@ -19,8 +19,7 @@ def classify_rust_type(type):
     return RustType.OTHER
 
 
-def summary_lookup(valobj, dict):
-    # type: (SBValue, dict) -> str
+def summary_lookup(valobj: lldb.SBValue, dict) -> str:
```

Comment:
I chose not to since there's no reason to ever use it - anything to dissuade people from trying seemed like a good idea. If you'd like i can add them though. Allegedly LLDB uses them for bookkeeping, but I haven't dug around the source code enough to know how/when they're used

Reply:
Actually wait, I checked and `dict` isn't actually a python dictionary

![image](https://github.com/user-attachments/assets/e144395f-113a-4bd3-ab0d-983b54d532cf)

Reply:
Huh, that is interesting. I wasn't sure what it would be but LLVM actually uses `: dict` in their repo a few times, thanks for double checking.

Maybe we could do a bit of both - what about something like `LldbOpaque = object` and then using `LldbOpaque` as the type hint? Then at least we have something there that also indicates it probably shouldn't be used.